doc/upgrading/ubuntu.md

```diff
@@ -21,7 +21,7 @@ sudo apt-get update
 
 If you are upgrading from OpenQuake Engine release 1.x, before you can process you have to run this command:
 ```bash
-sudo apt-get remove --purge python-oq.*
```

Comment:
python3-oq.* doesn't have to be here:

>If you are upgrading from OpenQuake Engine release 1.x, before you can process you have to run this command:

We should also change process to proceed